apps/voucher/graphql/schema.ts

```diff
@@ -0,0 +1,72 @@
+const typeDefs = `#graphql
+
+enum Status {
+  ACTIVE
+  PENDING 
+  PAID
+}
+
+type WithdrawLink {
+  id: ID!
+  userId: ID!
```

Comment:
what does the userId represents now?

Reply:
User who is creating the voucher

Reply:
does it map to the core UserId or core AccountId?

Reply:
core userid: `userData.me.id`